entrypoint.sh

```diff
@@ -7,11 +7,15 @@ if [ "$GITHUB_BASE_REF" ]; then
   echo "PR: fetching diff between origin/$GITHUB_BASE_REF and $GITHUB_SHA";
   git diff origin/"$GITHUB_BASE_REF" "$GITHUB_SHA" > $diff_filename;
 else
-  git fetch origin "$EVENT_BEFORE" --depth=1;
-  echo "PUSH: fetching diff between $EVENT_BEFORE and $GITHUB_SHA";
-  git diff "$EVENT_BEFORE" "$GITHUB_SHA" > $diff_filename;
+  if [ "$EVENT_BEFORE" = "0000000000000000000000000000000000000000" ]; then
+    echo "PUSH: fetching diff of initial commit";
+    git show --format="" "$GITHUB_SHA" > $diff_filename;
```

Comment:
What is the difference between empty format `""` and the `raw` format?

Reply:
The `format` flag formats the header information of the diff. The output for `raw` is below, and with `""`, everything above `diff --git ...` is removed, which is what our diff parser expects.
```
commit 4702b82e0aa73f41e1709d49b6e450ba6d785fd5
tree 1c0f766af11f19812608aa580c99347259ac678b
author Alan Yan <alan@nightfall.ai> 1596480615 -0400
committer Alan Yan <alan@nightfall.ai> 1596480615 -0400

    first commit

diff --git a/hi.txt b/hi.txt
new file mode 100644
index 0000000..c4ba990
--- /dev/null
+++ b/hi.txt
@@ -0,0 +1 @@
+hi my name is alan

```